nautobot_golden_config/templates/nautobot_golden_config/goldenconfigsetting.html

```diff
@@ -1,139 +1,247 @@
 {% extends 'base.html' %}
 {% load buttons %}
-{% load static %}
 {% load custom_links %}
 {% load helpers %}
-
-{% block title %}{{ object }}{% endblock %}
+{% load perms %}
+{% load plugins %}
+{% load static %}
+{% load tz %}
 
 {% block header %}
-<div class="pull-right noprint">
-    {% if perms.nautobot_golden_config.change_goldenconfigsetting %}
-    <a href="{% url 'plugins:nautobot_golden_config:goldenconfigsetting_edit' %}" class="btn btn-warning">
-        <span class="mdi mdi-pencil" aria-hidden="true"></span> Edit
-    </a>
-    {% endif %}
-</div>
-<h1>Golden Configuration Settings</h1>
-{% include 'inc/created_updated.html' %}
-<div class="pull-right noprint">
-    {% custom_links object %}
-</div>
-<ul class="nav nav-tabs">
-    <li role="presentation" {% if not active_tab %} class="active" {% endif %}>
-        <a href="{{ object.get_absolute_url }}">Settings</a>
-    </li>
-    {% if perms.extras.view_objectchange %}
-    <li role="presentation" {% if active_tab == 'changelog' %} class="active" {% endif %}>
-        <a href="{% url 'plugins:nautobot_golden_config:goldenconfigsetting_changelog' %}">Change Log</a>
-    </li>
-    {% endif %}
-</ul>
-{% endblock %}
+    <div class="row noprint">
```

Comment:
it's hard to check the HTML :) could you provide a picture in the PR description to show the look&feel?

Reply:
Will add this soon.